### PR TITLE
ref(api): Segment web urls

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Similar to https://github.com/getsentry/sentry/pull/13941

Mostly just for cleanliness purposes.

I also think we can probably get rid of a lot of the URLs that are just
generic react views, or at least keep them just for route names.

### CODE REVIEW

File: src/sentry/web/urls.py [hunk 1]
```diff
@@ -531,24 +544,15 @@ def init_all_applications():
         name='sentry-team-avatar-url'
     ),
 
-    # VSTS Marketplace extension install flow
```
Comment (EvanPurkhiser) [inline src/sentry/web/urls.py]: This moved down into the extensions group

**Review by markstory — APPROVED** [review]

**Comment by EvanPurkhiser** [comment]
Looks like acceptance tests may not be happy in this change.

**Comment by EvanPurkhiser** [comment]
Seems like there are some issues here. Will have to look closer.

**Comment by EvanPurkhiser** [comment]
Will come back to this later

**Pull request closed without merging**